bool NewTarget
int32 ControlGroup
---
bool isReachable
bool isAccepted
bool inProgress